src/v/kafka/server/group.h

```diff
@@ -599,7 +597,8 @@ class group {
     protocol_support _supported_protocols;
     member_map _members;
     int _num_members_joining;
-    absl::node_hash_set<kafka::member_id> _pending_members;
+    absl::node_hash_map<kafka::member_id, ss::timer<clock_type>>
```

Comment:
should this be a named struct. rather than a pair.

Reply:
it's not a pair it's map

Reply:
i was talking about the map's type. i.e.: `template<k, v> map = hash_set<std::pair<k,v>>` more or less. point being that the 

```
struct <some_name>{
   kafka::member_id id;
   ss::timer<clock_type> expiration_timeout;
};

```

or something like that. but it's a nit. so up to you.